.github/workflows/ci.yml

```diff
@@ -9,7 +9,7 @@ permissions:
   contents: read
 
 jobs:
-  build:
+  tests:
```

Comment:
I renamed this to test so it's clearer what's going on in the GitHub action list. It's now called "CI / tests" instead of "CI / build" and "Release / build"